src/controllers/scripting/controllerscriptenginebase.h

```diff
@@ -10,6 +10,8 @@
 #include "util/runtimeloggingcategory.h"
 #ifdef MIXXX_USE_QML
 #include "controllers/controllerenginethreadcontrol.h"
+#include "javascriptplayerproxy.h"
+#include "mixer/playermanager.h"
```

Comment:
Do we want to make this feature depending of `QML`? (MIXXX_USE_QML) I don't have an objection on this since we are now investing effort in getting QML out, so it will likely be enabled on 2.7.

Reply:
Yeah, probably not. This residual from when `QmlPlayerProxy` and `JavascriptPlayerProxy` where related.